Firestore/Source/Core/FSTFirestoreClient.mm

```diff
@@ -151,17 +150,17 @@ - (instancetype)initWithDatabaseInfo:(const DatabaseInfo &)databaseInfo
     // Defer initialization until we get the current user from the credentialChangeListener. This is
     // guaranteed to be synchronously dispatched onto our worker queue, so we will be initialized
     // before any subsequently queued work runs.
-    [_workerDispatchQueue dispatchAsync:^{
+    _workerQueue->Enqueue([=] {
```

Comment:
This lambda escapes the current stack frame so I'm wary of a default capture like this. I don't feel strongly though and likely non-harmful (default capture by reference would definitely not be OK).

I realize you've probably considered this, but is it worth adopting a convention of spelling out the capture when the lambda escapes?